README
see other documentation


Changes
Revision history for Map-Metro

{{$NEXT}}
 [API Changes]

 [Bug Fixes]

 [Documentation]

 [Enhancements]

 [New Features]

 [Other Changes]

 [Requirements]

 [Stats & Dependency Changes]

 [Testing]

0.2405 2016-07-23T11:33:37Z UTC
 [Requirements]
 - Bump Moose version to ensure Type::Tiny compatibility (fixes #12).

 [Stats & Dependency Changes]
 - 28 files changed, 28 insertions(+), 30 deletions(-)
 - (run req) ~ Moose 2.1405 --> 2.1803
 - (dev sug) ~ Dist::Iller::Config::Author::CSSON 0.0309 --> 0.0310

0.2404 2016-04-30T17:20:01Z UTC
 [New Features]
 - Added experimental to_hash() method to several of the graph classes.

 [Stats & Dependency Changes]
 - 30 files changed, 158 insertions(+), 29 deletions(-)
 - (run req) + PerlX::Maybe 1.001
 - (run req) ~ List::Util 1.42 --> 1.45
 - (dev req) + Dist::Zilla::Plugin::MetaConfig (any)

0.2403 2016-03-19T20:00:19Z UTC
 [Bug Fixes]
 - Add missing use statement (#11).

 [Stats & Dependency Changes]
 - 28 files changed, 29 insertions(+), 28 deletions(-)

0.2402 2016-02-27T12:42:39Z UTC
 [Requirements]
 - Minimum version required for MooseX::App due to app_description.

 [Stats & Dependency Changes]
 - 30 files changed, 31 insertions(+), 31 deletions(-)
 - (run req) ~ MooseX::App (any) --> 1.29

 [Testing]
 - Use shorter non-existing line id to avoid some systems using scientific
   notation.

0.2401 2016-02-25T00:35:45Z UTC
 [Documentation]
 - Improved some documentation wording.

 [Other Changes]
 - Removed no longer used dependency.

 [Stats & Dependency Changes]
 - 29 files changed, 43 insertions(+), 52 deletions(-)
 - (run req) - MooseX::SetOnce

0.2400 2016-02-24T18:56:43Z UTC
 [Bug Fixes]
 - The 'available' command no longer lists <map>::Lines as available map
   names.

 [Enhancements]
 - Lowered required Perl to 5.10 (from 5.16). If Perl version is 5.16 or
   newer, 'fc' is used for string comparisons instead of 'lc'.

 [Other Changes]
 - Rewritten to no longer use Moops/Kavorka.

 [Stats & Dependency Changes]
 - 41 files changed, 2211 insertions(+), 2040 deletions(-)
 - (run req) + Safe::Isa 1.000005
 - (run req) + Throwable::SugarFactory (any)
 - (run req) + Try::Tiny 0.24
 - (run req) + namespace::autoclean 0.22
 - (run req) ~ File::ShareDir (any) --> 1.102
 - (run req) ~ Graph (any) --> 0.9704
 - (run req) ~ List::Util 1.33 --> 1.42
 - (run req) ~ Moose (any) --> 2.1405
 - (run req) ~ MooseX::AttributeDocumented (any) --> 0.1003
 - (run req) ~ MooseX::AttributeShortcuts (any) --> 0.028
 - (run req) ~ Path::Tiny (any) --> 0.072
 - (run req) ~ Types::Path::Tiny (any) --> 0.005
 - (run req) ~ Types::Standard (any) --> 1.000005
 - (run req) ~ perl 5.016000 --> 5.010000
 - (run req) - Kavorka
 - (run req) - Kavorka::TraitFor::Parameter::doc
 - (run req) - Moops
 - (run req) - MoopsX::UsingMoose
 - (run req) - Sub::Exporter
 - (run req) - Syntax::Collector
 - (run req) - Throwable::X
 - (run req) - Type::Tiny::Enum
 - (run req) - namespace::clean
 - (run req) - true
 - (run rec) - GraphViz2
 - (test req) + IO::Handle (any)
 - (test req) + IPC::Open3 (any)
 - (dev req) + Badge::Depot 0.0103
 - (dev req) + Badge::Depot::Plugin::Coverage (any)
 - (dev req) + Badge::Depot::Plugin::Cpantesters (any)
 - (dev req) + Badge::Depot::Plugin::Kwalitee (any)
 - (dev req) + Badge::Depot::Plugin::Perl (any)
 - (dev req) + Badge::Depot::Plugin::Travis (any)
 - (dev req) + Dist::Zilla 5.000
 - (dev req) + Dist::Zilla::Plugin::Authority 1.009
 - (dev req) + Dist::Zilla::Plugin::BumpVersionAfterRelease::Transitional
   0.007
 - (dev req) + Dist::Zilla::Plugin::ChangeStats::Dependencies::Git 0.0100
 - (dev req) + Dist::Zilla::Plugin::Git 2.036
 - (dev req) + Dist::Zilla::Plugin::Git::CheckFor::CorrectBranch 0.013
 - (dev req) + Dist::Zilla::Plugin::Git::Commit (any)
 - (dev req) + Dist::Zilla::Plugin::NextRelease::Grouped 0.0200
 - (dev req) + Dist::Zilla::Plugin::RewriteVersion::Transitional 0.007
 - (dev req) + Dist::Zilla::Plugin::Test::CleanNamespaces (any)
 - (dev req) + Dist::Zilla::Plugin::Test::Compile 2.054
 - (dev req) + Dist::Zilla::Plugin::Test::EOF 0.0501
 - (dev req) + Dist::Zilla::Plugin::Test::EOL 0.18
 - (dev req) + Dist::Zilla::Plugin::Test::Kwalitee::Extra v0.2.1
 - (dev req) + Dist::Zilla::Plugin::Test::NoTabs 0.15
 - (dev req) + Pod::Weaver::Section::GenerateSection 1.01
 - (dev req) + Test::CleanNamespaces 0.15
 - (dev req) + Test::EOF (any)
 - (dev req) + Test::EOL (any)
 - (dev req) + Test::More 0.88
 - (dev req) + Test::NoTabs 1.4
 - (dev req) ~ Dist::Zilla::Plugin::CheckChangesHasContent (any) --> 0.008
 - (dev req) ~ Dist::Zilla::Plugin::Clean (any) --> 0.07
 - (dev req) ~ Dist::Zilla::Plugin::CopyFilesFromBuild (any) --> 0.151680
 - (dev req) ~ Dist::Zilla::Plugin::Git::Contributors (any) --> 0.017
 - (dev req) ~ Dist::Zilla::Plugin::GithubMeta (any) --> 0.54
 - (dev req) ~ Dist::Zilla::Plugin::InstallRelease (any) --> 0.008
 - (dev req) ~ Dist::Zilla::Plugin::MetaProvides::Package (any) -->
   2.003001
 - (dev req) ~ Dist::Zilla::Plugin::PodWeaver (any) --> 4.006
 - (dev req) ~ Dist::Zilla::Plugin::ReadmeAnyFromPod (any) --> 0.150250
 - (dev req) ~ Dist::Zilla::Plugin::RunExtraTests (any) --> 0.028
 - (dev req) ~ Dist::Zilla::Plugin::Test::ReportPrereqs (any) --> 0.021
 - (dev req) ~ Dist::Zilla::Plugin::TravisYML (any) --> 1.15
 - (dev req) ~ Pod::Elemental::Transformer::Splint (any) --> 0.1201
 - (dev req) ~ Pod::Weaver::Section::Badges (any) --> 0.0401
 - (dev req) ~ Test::Warnings (any) --> 0.026
 - (dev req) - Dist::Zilla::Plugin::CheckExtraTests
 - (dev req) - Dist::Zilla::Plugin::MetaProvides::Class
 - (dev req) - Dist::Zilla::Plugin::NextRelease
 - (dev req) - Dist::Zilla::Plugin::NextVersion::Semantic
 - (dev req) - Dist::Zilla::Plugin::OurPkgVersion
 - (dev req) - Dist::Zilla::Plugin::PodnameFromClassname
 - (dev req) - Dist::Zilla::Plugin::Prereqs::Plugins
 - (dev req) - Dist::Zilla::Plugin::PreviousVersion::Changelog
 - (dev req) - Dist::Zilla::Plugin::ReversionOnRelease

Makefile.PL
# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v6.006.
use strict;
use warnings;

use 5.010000;

use ExtUtils::MakeMaker;

my %WriteMakefileArgs = (
  "ABSTRACT" => "Public transport graphing",
  "AUTHOR" => "Erik Carlsson <info\@code301.com>",
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0
  },
  "DISTNAME" => "Map-Metro",
  "LICENSE" => "perl",
  "MIN_PERL_VERSION" => "5.010000",
  "NAME" => "Map::Metro",
  "PREREQ_PM" => {
    "Eponymous::Hash" => 0,
    "File::ShareDir" => "1.102",
    "Graph" => "0.9704",
    "IO::Interactive" => 0,
    "List::Compare" => 0,
    "List::Util" => "1.45",
    "Module::Pluggable" => 0,
    "Moose" => "2.1803",
    "MooseX::App" => "1.29",
    "MooseX::AttributeDocumented" => "0.1003",
    "MooseX::AttributeShortcuts" => "0.028",
    "Path::Tiny" => "0.072",
    "PerlX::Maybe" => "1.001",
    "Safe::Isa" => "1.000005",
    "String::Trim" => 0,
    "Term::Size::Any" => 0,
    "Text::Undiacritic" => 0,
    "Throwable::SugarFactory" => 0,
    "Try::Tiny" => "0.24",
    "Types::Path::Tiny" => "0.005",
    "Types::Standard" => "1.000005",
    "XML::Writer" => 0,
    "namespace::autoclean" => "0.22"
  },
  "TEST_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0,
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::More" => "0.96"
  },
  "VERSION" => "0.2406",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


my %FallbackPrereqs = (
  "Eponymous::Hash" => 0,
  "ExtUtils::MakeMaker" => 0,
  "File::ShareDir" => "1.102",
  "File::Spec" => 0,
  "Graph" => "0.9704",
  "IO::Handle" => 0,
  "IO::Interactive" => 0,
  "IPC::Open3" => 0,
  "List::Compare" => 0,
  "List::Util" => "1.45",
  "Module::Pluggable" => 0,
  "Moose" => "2.1803",
  "MooseX::App" => "1.29",
  "MooseX::AttributeDocumented" => "0.1003",
  "MooseX::AttributeShortcuts" => "0.028",
  "Path::Tiny" => "0.072",
  "PerlX::Maybe" => "1.001",
  "Safe::Isa" => "1.000005",
  "String::Trim" => 0,
  "Term::Size::Any" => 0,
  "Test::More" => "0.96",
  "Text::Undiacritic" => 0,
  "Throwable::SugarFactory" => 0,
  "Try::Tiny" => "0.24",
  "Types::Path::Tiny" => "0.005",
  "Types::Standard" => "1.000005",
  "XML::Writer" => 0,
  "namespace::autoclean" => "0.22"
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);
